sky/tools/sky_snapshot/main.cc

```diff
@@ -63,5 +79,13 @@ int main(int argc, const char* argv[]) {
   CHECK(command_line.HasSwitch(switches::kSnapshot)) << "Need --snapshot";
   WriteSnapshot(command_line.GetSwitchValuePath(switches::kSnapshot));
 
+  if (command_line.HasSwitch(switches::kDepfile)) {
+    CHECK(command_line.HasSwitch(switches::kBuildOutput)) <<
+        "Need --build-output with --depfile";
+    WriteDependencies(command_line.GetSwitchValuePath(switches::kDepfile),
+                      command_line.GetSwitchValueASCII(switches::kBuildOutput),
```

Comment:
Rather than crashing when `--build-output` is missing, maybe have it default to the path where we wrote the snapshot?